elasticsearch/json-ingest-5plus.yml

```diff
@@ -0,0 +1,51 @@
+type: json
+data:
+  url: /_nodes/_local/stats/indices,transport,http,thread_pool,jvm,process,fs,script,discovery,ingest,adaptive_selection?format=smile
+  server: ${ST_MONITOR_ES_NODE_HOSTPORT}
+  basicHttpAuthUsername: ${ST_MONITOR_ES_NODE_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${ST_MONITOR_ES_NODE_BASICAUTH_PASSWORD}
+  smileFormat: true
+require:
+  - className: com.sematext.spm.client.es.EsVersionCheck
+    values: 5-*
+
+observation:
+  - name: ingest_stats
+    metricNamespace: es
+    path: $.nodes.${nodeId}.ingest.pipelines.${pipeline}
+
+    metric:
+      - name: ingest.count.total
+        source: count
+        type: counter
+        label: ingest count
+        description: number calls to this pipeline
```

Comment:
Missing `of`, same as for the metric below.

Reply:
Oops. Fixed now, thanks for the catch!